src/AnyChart/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="AnyChart" version="1.3.5" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="AnyChart" version="2.0.1" xmlns="http://www.mendix.com/clientModule/1.0/">
```

Comment:
This seems like quite the jump, is this correct? (Major bump + patch bump?)

Reply:
@jordanch the widget version doesnt need to match the module for any chart version, this is not right as Chak said :)

Reply:
Thanks for the catch

Reply:
@Keraito @diego-antonelli when responding to this comment, I forgot that a v2.0.0 of AnyChart has already been released to the marketplace. This explains why I bumped from 1.3.5 to 2.0.1. I guess whoever released the widget forgot to update this version.